torchrec/sparse/jagged_tensor.py

```diff
@@ -876,6 +876,80 @@ def empty_like(kjt: "KeyedJaggedTensor") -> "KeyedJaggedTensor":
             stride=kjt.stride(),
         )
 
+    @staticmethod
+    def from_jt_dict(jt_dict: Dict[str, JaggedTensor]) -> "KeyedJaggedTensor":
+        """
+        Constructs a KeyedJaggedTensor from a Dict[str, JaggedTensor],
+        but this function will ONLY work if the JaggedTensors all
+        have the same "implicit" batch_size dimension.
+
+        Basically, we can visualize JaggedTensors as 2-D tensors
+        of the format of [batch_size x variable_feature_dim].
+        In case, we have some batch without a feature value,
+        the input JaggedTensor could just not include any values.
+
+        But KeyedJaggedTensor (by default) typically pad "None"
+        so that all the JaggedTensors stored in the KeyedJaggedTensor
+        have the same batch_size dimension. That is, in the case,
+        the JaggedTensor input didn't automatically pad
+        for the empty batches, this function would error / not work.
+
+        Consider the visualization of the following KeyedJaggedTensor:
+        #              0       1        2  <-- dim_1
+        # "Feature0"   [V0,V1] None    [V2]
+        # "Feature1"   [V3]    [V4]    [V5,V6,V7]
+        #   ^
+        #  dim_0
+
+        Notice that the inputs for this KeyedJaggedTensor would have looked like:
+            values: torch.Tensor = [V0, V1, V2, V3, V4, V5, V6, V7]  # V == any tensor datatype
+            weights: torch.Tensor = [W0, W1, W2, W3, W4, W5, W6, W7]  # W == any tensor datatype
+            lengths: torch.Tensor = [2, 0, 1, 1, 1, 3]  # representing the jagged slice
+            offsets: torch.Tensor = [0, 2, 2, 3, 4, 5, 8]  # offsets from 0 for each jagged slice
+            keys: List[str] = ["Feature0", "Feature1"]  # correspond to each value of dim_0
+            index_per_key: Dict[str, int] = {"Feature0": 0, "Feature1": 1}  # index for each key
+            offset_per_key: List[int] = [0, 3, 8]  # start offset for each key and final offset
```

Comment:
can you remove weights, offsets, index_per_key, offset_per_key ? seems not essential for this method